klippy/kinematics/delta.py

```diff
@@ -150,6 +150,12 @@ def move(self, print_time, move):
             self._check_motor_enable(print_time)
         for rail in self.rails:
             rail.step_itersolve(move.cmove)
+    def get_status(self):
+        return { 'homed': tuple([self.need_home] * 3),
+                 'limits': tuple([(-sqrt(self.limit_xy2),
+                                   sqrt(self.limit_xy2))] * 2 +
+                                 [(self.min_z, self.limit_z)])
```

Comment:
As mentioned earlier, I'm not sure exporting the "limits" makes sense - it's not really intended to be used outside of the class.  It also doesn't have much meaning on some kinematics.